### PR TITLE
Pod code and tests

### PR DESCRIPTION
**Pull request by dostrander**
- Add's the RZintrinsicContentSizeTextView to the pod
- Add the project Pods
- Add tests for RZintrinsicContentSizeTextView

### CODE REVIEW

**Comment by dostrander** [comment]
@jatraiz can you take a look at this. This is a majority of my pod code. It's so big because I had to checkin the podfiles too

**Comment by alexrrouse** [comment]
@dostrander I would recommend breaking the dependency on RZUtils if possible.

File: Pod/Categories/UIView+RZAutoLayoutReplacementHelpers.m [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+- (NSLayoutConstraint *)rz_replacePinnedHeightConstraintWithNewPinnedHeight:(CGFloat)height priority:(CGFloat)priority
+{
+    NSLayoutConstraint *rz_pinnedHeightConstraint = self.rz_pinnedHeightConstraint;
+
+    if ( rz_pinnedHeightConstraint ) {
+        [self removeConstraint:rz_pinnedHeightConstraint];
+    }
+
+    return [self rz_pinHeightTo:height priority:priority];
+
```
Comment (jatraiz) [inline Pod/Categories/UIView+RZAutoLayoutReplacementHelpers.m:48]: extra line break

File: Pod/Classes/RZIntrinsicContentSizeTextView.h [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+/**
+ *  Set this to have attributed placeholder text
+ */
+@property (copy, nonatomic, readwrite) NSAttributedString *attributedPlaceholder;
+
+/**
+ *  Delegate to have more fine tuned control for if/when/how it should change it's size.
+ *  @note Many in many instances you won't have to use this.
+ */
```
Comment (jatraiz) [inline Pod/Classes/RZIntrinsicContentSizeTextView.h:64]: typo: Many in many

File: Pod/Classes/RZIntrinsicContentSizeTextView.m [hunk 3]
```diff
@@ -0,0 +1,414 @@
...
+@synthesize rz_minHeightConstraint = _rz_minHeightConstraint;
+@synthesize rz_maxHeightConstraint = _rz_maxHeightConstraint;
+
+#pragma mark - View Life Cycle
+
+- (id)initWithFrame:(CGRect)frame textContainer:(NSTextContainer *)textContainer
+{
+    self = [super initWithFrame:frame textContainer:textContainer];
+    if ( self ) {
+        [self configureIntrinsicTextView];
```
Comment (jatraiz) [inline Pod/Classes/RZIntrinsicContentSizeTextView.m:70]: back yesterdays discussion for these ini: if ( self != nil )

File: Pod/Classes/RZIntrinsicContentSizeTextView.m [hunk 4]
```diff
@@ -0,0 +1,414 @@
...
+}
+
+#pragma mark - Super Setter Overrides
+
+- (void)setTextContainerInset:(UIEdgeInsets)textContainerInset
+{
+    [super setTextContainerInset:textContainerInset];
+    [self adjustPlaceholderPosition];
+    [self scrollToBottom];
+}
```
Comment (jatraiz) [inline Pod/Classes/RZIntrinsicContentSizeTextView.m:154]: is it always going to the case that the once setting the inset scrollToBottom should be called?

File: Pod/Classes/RZIntrinsicContentSizeTextView.m [hunk 5]
```diff
@@ -0,0 +1,414 @@
...
+
+- (void)setText:(NSString *)text
+{
+    [super setText:text];
+    [self adjustPlaceholderForTextChange];
+    [self adjustHeightIfNeededAnimated:self.shouldAnimateSizeChange];
+
```
Comment (jatraiz) [inline Pod/Classes/RZIntrinsicContentSizeTextView.m:161]: line break

**Pull request closed without merging**